.github/workflows/pythonapp.yml

```diff
@@ -146,12 +146,21 @@ jobs:
     strategy:
       matrix:
         environment:
+          - "PT14+CUDA100"
```

Comment:
I think maybe PT14+NV20.03 is enough for PyTorch 1.4? Don't want the PR tests to run too slow..
Thanks.

Reply:
actually I'm not sure whether NV20.03's pytorch is exactly the same as the pip installed one...I think it's good to have this test. But, for the overall test speed, perhaps we could think about dropping the PT1.4 support?  it doesn't have the AMP feature

Reply:
@IsaacYangSLA @ericspod , what do you guys think? Do you agree to drop PyTorch 1.4 support directly?

Thanks.

Reply:
I'm good with dropping it.

Reply:
@IsaacYangSLA , what do you think?
If all you guys think it's OK to drop PyTorch 1.4 support, @wyli could you please help update this PR to drop PyTorch 1.4 directly?

Thanks.

Reply:
happy to make the change, perhaps we could discuss this early next week? this is kind of related to https://github.com/Project-MONAI/MONAI/issues/233

Reply:
NV PyTorch containers 19.11~20.02 were based on PT1.4.  Since pytorch:20.08 and 20.09 are PT1.7, I think dropping PT1.4 support is OK.  If we really need PT14 support, we could choose nvcr.io/nvidia/pytorch:20.03 (build with CUDA 10.2).  All the release and pt versions come from this page https://docs.nvidia.com/deeplearning/frameworks/pytorch-release-notes/index.html.

Reply:
Hi @wyli ,

Could you please help update this PR to drop PT1.4 directly?
I remember we have some logic in interplate transforms and determinism API to decide PyTorch version 1.4 or 1.5, please also remove that.

Thanks.

Reply:
One question, why are we using Nvidia's Pytorch containers? We can use Pytorch's containers for specific Pytorch versions, or just a CUDA container and install Anaconda through the Dockerfile. I don't think right now using a container with Pytorch 1.7 is a good idea since it isn't officially released yet.